dask_kubernetes/helm.py

```diff
@@ -251,20 +252,31 @@ def scale(self, n_workers):
         >>> cluster.scale(4)
         >>> cluster
         HelmCluster('tcp://localhost:8786', workers=4, threads=24, memory=24.96 GB)
-
+        >>> cluster.scale(5, worker_group = "high-mem-worker")
+        >>> cluster
+        # TODO: Finish scaling example
```

Comment:
Do you mind if I update all the examples, so it will look consistent?

Reply:
Sure